src/redux.js

```diff
@@ -92,20 +90,12 @@ export function setSelectedTrainer(selectedTrainer) {
   return { type: SET_SELECTED_TRAINER, selectedTrainer };
 }
 
-export function setKValue(kValue) {
-  return { type: SET_K_VALUE, kValue };
-}
-
 export const setColumnsByDataType = (column, dataType) => ({
   type: SET_COLUMNS_BY_DATA_TYPE,
   column,
   dataType
 });
 
-export function setSelectedFeatures(selectedFeatures) {
-  return { type: SET_SELECTED_FEATURES, selectedFeatures };
-}
-
```

Comment:
Similar to above, can you remove the action types references for each of these that you're deleting, please.

Reply:
Good catch, will do.